can-param.js

```diff
@@ -21,11 +23,36 @@ function buildParam(prefix, obj, add) {
 module.exports = namespace.param = function param(object) {
 	var pairs = [],
 		add = function (key, value) {
+			value = standardsMode && value == null ? '' : value;
 			pairs.push(encodeURIComponent(key) + '=' + encodeURIComponent(value));
 		};
 	for (var name in object) {
-		buildParam(name, object[name], add);
+		if (!standardsMode || typeof object[name] !== 'undefined') {
+			buildParam(name, object[name], add);
+		}
 	}
 	return pairs.join('&')
 		.replace(/%20/g, '+');
 };
+
+/**
+ * @function can-param.setStandardsMode setStandardsMode
+ * @parent can-param.methods
+ * @description Set whether to treat null and undefined specially when serializing
+ * 
+ * @signature `param.setStandardsMode(boolean)`
+ *
+ * Set whether to serialize values in a manner compliant with jQuery and URLSearchParameters, or to use the classic
```

Comment:
Might be good to link to https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams. At least want to say `URLSearchParams` instead of `URLSearchParameters`.